configs/platforms/ubuntu-14.04-i386.rb

```diff
@@ -1,4 +1,3 @@
-peversion = IO.read('peversion').strip
```

Comment:
Appears to be unused. Looks like it pointing to enterprise repo was taken out with https://github.com/puppetlabs/bolt-vanagon/commit/1e1170a89ab3329a8dfad104c5fc2c293b9d80e9#diff-2994752a933486f31ab928ba239d6385 but the variable assignment was not removed with it.